tests/test_api.py

```diff
@@ -1,10 +1,28 @@
-import os
+import cads_api_client.legacy_api_client
+import pytest
 
 import cdsapi
 
 
-def test_request():
-    c = cdsapi.Client()
+@pytest.mark.parametrize(
+    "url,key,expected_client",
+    [
+        (
+            None,
+            None,
+            cdsapi.Client,
+        ),
+        (
+            "http://cds2-test.copernicus-climate.eu/api",
+            "00000000-0000-4000-a000-000000000000",
```

Comment:
Maybe this test should use secrets

Reply:
Fine by me, but we (B-Open) do not have the permissions to add secrets.
Also, GH secrets would not work when pytest is executed locally.